man/htmlFieldset.Rd

```diff
@@ -73,3 +73,30 @@ those elements have the following types:
   - prop_name (character; optional): holds which property is loading
   - component_name (character; optional): holds the name of the component that is loading. Object that holds the loading state object coming from dash-renderer}
 }
+\examples{
+\dontrun{
+
+library(dash)
+library(dashHtmlComponents)
+library(dashCoreComponents)
+
+app <- Dash$new()
+
+app$layout(htmlDiv(list(
+  htmlFieldset(
+    children = list('Choose your favourite Dash HTML component',
```

Comment:
Can't say we do this systematically, but do seem (tend) to default to English-US spelling.
```suggestion
    children = list('Choose your favorite Dash HTML component',
```